src/main/resources/db/migration/V0001__init.sql

```diff
@@ -0,0 +1,481 @@
+CREATE INDEX "IR_BULK_REFERENCE_I" ON "SDT_OWNER"."INDIVIDUAL_REQUESTS" ("SDT_BULK_REFERENCE");
+CREATE INDEX "IR_BULK_SUBMISSION_ID_I" ON "SDT_OWNER"."INDIVIDUAL_REQUESTS" ("BULK_SUBMISSION_ID");
+CREATE INDEX "IR_LWR_CUSTOMER_REQ_REF_CD_I" ON "SDT_OWNER"."INDIVIDUAL_REQUESTS" (LOWER("CUSTOMER_REQUEST_REF"), "CREATED_DATE");
+CREATE INDEX "IR_SDT_REQUEST_REFERENCE_I" ON "SDT_OWNER"."INDIVIDUAL_REQUESTS" ("SDT_REQUEST_REFERENCE");
+CREATE UNIQUE INDEX "SERVICE_REQUESTS_PK" ON "SDT_OWNER"."SERVICE_REQUESTS" ("SERVICE_REQUEST_ID");
+CREATE UNIQUE INDEX "SERVICE_ROUTINGS_PK" ON "SDT_OWNER"."SERVICE_ROUTINGS" ("SERVICE_ROUTINGS_ID");
+CREATE UNIQUE INDEX "SERVICE_TYPES_PK" ON "SDT_OWNER"."SERVICE_TYPES" ("SERVICE_TYPE_ID");
+CREATE INDEX "SR_SERVICE_TYPE_I" ON "SDT_OWNER"."SERVICE_ROUTINGS" ("SERVICE_TYPE_ID");
+CREATE INDEX "SR_TARGET_APPLICATION_I" ON "SDT_OWNER"."SERVICE_ROUTINGS" ("TARGET_APPLICATION_ID");
+CREATE INDEX "ST_SERVICE_TYPE_NAME" ON "SDT_OWNER"."SERVICE_TYPES" ("SERVICE_TYPE_NAME");
+CREATE UNIQUE INDEX "TARGET_APPLICATIONS_PK" ON "SDT_OWNER"."TARGET_APPLICATIONS" ("TARGET_APPLICATION_ID");
+CREATE INDEX "TA_TARGET_APPLICATION_NAME" ON "SDT_OWNER"."TARGET_APPLICATIONS" ("TARGET_APPLICATION_NAME");
+
+------------------------------------------------
+-- Create Primary Keys for SDT_OWNER
+------------------------------------------------
+ALTER TABLE "SDT_OWNER"."BULK_CUSTOMERS" ADD CONSTRAINT "BULK_CUSTOMERS_PK" PRIMARY KEY USING INDEX "BULK_CUSTOMERS_PK";
+ALTER TABLE "SDT_OWNER"."BULK_CUSTOMER_APPLICATIONS" ADD CONSTRAINT "BULK_CUSTOMER_APPLICATIONS_PK" PRIMARY KEY USING INDEX "BULK_CUSTOMER_APPLICATIONS_PK";
+ALTER TABLE "SDT_OWNER"."BULK_SUBMISSIONS" ADD CONSTRAINT "BULK_SUBMISSIONS_PK" PRIMARY KEY USING INDEX "BULK_SUBMISSIONS_PK";
+ALTER TABLE "SDT_OWNER"."ERROR_LOGS" ADD CONSTRAINT "ERROR_LOGS_PK" PRIMARY KEY USING INDEX "ERROR_LOGS_PK";
+ALTER TABLE "SDT_OWNER"."ERROR_MESSAGES" ADD CONSTRAINT "ERROR_MESSAGES_PK" PRIMARY KEY USING INDEX "ERROR_MESSAGES_PK";
+ALTER TABLE "SDT_OWNER"."GLOBAL_PARAMETERS" ADD CONSTRAINT "GLOBAL_PARAMETERS_PK" PRIMARY KEY USING INDEX "GLOBAL_PARAMETERS_PK";
+ALTER TABLE "SDT_OWNER"."INDIVIDUAL_REQUESTS" ADD CONSTRAINT "INDIVIDUAL_REQUESTS_PK" PRIMARY KEY USING INDEX "INDIVIDUAL_REQUESTS_PK";
+ALTER TABLE "SDT_OWNER"."SERVICE_REQUESTS" ADD CONSTRAINT "SERVICE_REQUESTS_PK" PRIMARY KEY USING INDEX "SERVICE_REQUESTS_PK";
+ALTER TABLE "SDT_OWNER"."SERVICE_ROUTINGS" ADD CONSTRAINT "SERVICE_ROUTINGS_PK" PRIMARY KEY USING INDEX "SERVICE_ROUTINGS_PK";
+ALTER TABLE "SDT_OWNER"."SERVICE_TYPES" ADD CONSTRAINT "SERVICE_TYPES_PK" PRIMARY KEY USING INDEX "SERVICE_TYPES_PK";
+ALTER TABLE "SDT_OWNER"."TARGET_APPLICATIONS" ADD CONSTRAINT "TARGET_APPLICATIONS_PK" PRIMARY KEY USING INDEX "TARGET_APPLICATIONS_PK";
+
+------------------------------------------------
+-- Create Check Constraints for SDT_OWNER
+------------------------------------------------
+-- capitalised names reqd?
```

Comment:
Don't forget to remove this comment when done.

Reply:
Done